File: Android_App_Java_Gucci_Store/app/src/main/java/com/example/dhura_mistry_project2/CartItems.java
package com.example.dhura_mistry_project2;

public class CartItems {

    private String itemImage;
    private String itemName;
    private String itemPrice;
    private int quantity;
    private int totalPrice;
    String documentId;

    public CartItems() {
    }

    public CartItems(String itemImage, String itemName, String itemPrice, int quantity, int totalPrice) {
        this.itemImage = itemImage;
        this.itemName = itemName;
        this.itemPrice = itemPrice;
        this.quantity = quantity;
        this.totalPrice = totalPrice;
    }

    public String getDocumentId() {
        return documentId;
    }

    public void setDocumentId(String documentId) {
        this.documentId = documentId;
    }

    public String getItemImage() {
        return itemImage;
    }

    public void setItemImage(String itemImage) {
        this.itemImage = itemImage;
    }

    public String getItemName() {
        return itemName;
    }

    public void setItemName(String itemName) {
        this.itemName = itemName;
    }

    public String getItemPrice() {
        return itemPrice;
    }

    public void setItemPrice(String itemPrice) {
        this.itemPrice = itemPrice;
    }

    public int getQuantity() {
        return quantity;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }

    public int getTotalPrice() {
        return totalPrice;
    }

    public void setTotalPrice(int totalPrice) {
        this.totalPrice = totalPrice;
    }
}

File: Kotlin_App_1/app/src/main/java/com/example/group1_project1/CartItems.kt
package com.example.group1_project1

class CartItems {
    var itemImage: String? = null
    var itemName: String? = null
    var itemPrice: String? = null
    var quantity = 0
    var totalPrice = 0
    var documentId: String? = null

    constructor() {}
    constructor(
        itemImage: String?,
        itemName: String?,
        itemPrice: String?,
        quantity: Int,
        totalPrice: Int
    ) {
        this.itemImage = itemImage
        this.itemName = itemName
        this.itemPrice = itemPrice
        this.quantity = quantity
        this.totalPrice = totalPrice
    }
}
